invenio_records_files/api.py

```diff
@@ -224,18 +316,24 @@ class FilesMixin(object):
     """
 
     file_cls = FileObject
+
```

Comment:
I was not adding it because by default it is generating an "alias of class" comment with the correct link. Should I add a custom comment?
![aliasof](https://cloud.githubusercontent.com/assets/6481083/17440375/97d870e6-5b2c-11e6-8224-f4138cacf569.png)